docs/standard/garbage-collection/fundamentals.md

```diff
@@ -166,7 +166,7 @@ ms.author: "ronpet"
   
  The garbage collector uses the following information to determine whether objects are live:  
   
--   **Stack roots**. Stack variables provided by the just-in-time (JIT) compiler and stack walker.  
+-   **Stack roots**. Stack variables provided by the just-in-time (JIT) compiler and stack walker. Note that JIT optimizations can lengthen and shorten region of code within which the stack variable is reported to the garbage collector.
```

Comment:
Both optimizations and debug flags, right?

Also nit: "region of code" => "regions of code"